src/System.Data.Common/tests/System/Data/Common/RowUpdatedEventArgsTest.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Xunit;
+
+namespace System.Data.Common
+{
+    public class RowUpdatedEventArgsTest
+    {
+        [Fact]
+        public void ConstructorFail()
```

Comment:
We generally try to use slightly more descriptive names, like `Ctor_InvalidArgument_ThrowsArgumentOutOfRangeException`.